client/src/Profile.css

```diff
@@ -0,0 +1,291 @@
+  font-size: 18px;
+  border-radius: 50%;
+}
+
+.profPicContainer:hover .editPic {
+  display: inline;
+}
+
+.profPicContainer:hover .profilePic {
+  opacity: 0.3;
+}
+
+.nameContainer {
+  display: flex;
+  flex-direction: column;
+  font-family: 'Montserrat', sans-serif;
+  text-align: left;
+  padding-left: 12px;
+  padding-top: 8px;
+  margin-right: 20px;
+}
+
+.tagsContainer {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin: 12px 20px 22px 20px;
+  max-width: 500px;
+}
+
+.profileTextContainer {
+  font: 25px 'Montserrat', sans-serif;
+  color: #00849c;
+  display: flex;
+  flex-direction: column;
+  margin-left: 15px;
+  margin-right: 20px;
+}
+
+.editProfileTextContainer {
+  font: 25px 'Montserrat', sans-serif;
+  color: #00849c;
+  display: flex;
+  flex-direction: row;
+  width: 80%;
+  flex-wrap: wrap;
+  margin-left: 15px;
+}
+
+.CoOpLocation {
+  font: 15px 'Montserrat', sans-serif;
+  font-weight: 500;
+  line-height: 18px;
+  color: black;
+  padding: 5px;
+}
+
+.CoOpContact {
+  font: 12px 'Montserrat', sans-serif;
+  color: black;
+  font-weight: 500;
+  line-height: 15px;
+  padding-left: 5px;
+  margin-top: 3px;
+}
+
+/*Note: I am changing all link colors to red*/
+a:visited {
+  background-color: transparent;
+  color: #c21d49;
+}
+
+.NameInput {
+  font: 25px 'Montserrat', sans-serif;
+  font-weight: bold;
+  color: #00849c;
+  height: 30px;
+  width: 294px;
+  padding: 10px;
+  outline: none;
+}
+.smallInput {
+  font: 11px 'Montserrat', sans-serif;
+  font-weight: 500;
+  line-height: 11px;
+  border-radius: 5px;
+  color: black;
+  padding: 5px;
+  padding-left: 8px;
+  padding-right: 8px;
+  width: 126px;
+  height: 15px;
+  outline: none;
+}
+
+.topDiv {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-top: 10px;
+  width: 100%;
+}
+
+.logoDiv {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  width: 50%;
+  padding-left: 10px;
+}
+
+.editDiv {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  width: 50%;
+  padding-right: 20px;
+}
+
+.logo {
+  width: 112px;
+  height: 40px;
+  padding-top: 10px;
+}
+
+.editButton {
+  width: 49px;
+  height: 20px;
+  border-radius: 28px;
+  background-color: #f8f9fb;
+  font: bold 10px 'Montserrat', sans-serif;
+  color: #81a1b7;
+  outline: none;
+  border: none;
+  cursor: pointer;
+  box-shadow: 0 0px 5px 0 rgba(0, 0, 0, 0.2), 0 0px 5px 0 rgba(0, 0, 0, 0.19);
+}
+
+.saveButton {
+  width: 49px;
+  height: 20px;
+  border-radius: 28px;
+  background-color: #81a1b7;
+  font: bold 10px 'Montserrat', sans-serif;
+  color: white;
+  outline: none;
+  border: none;
+  cursor: pointer;
+  box-shadow: 0 0px 5px 0 rgba(0, 0, 0, 0.2), 0 0px 5px 0 rgba(0, 0, 0, 0.19);
+}
+
+.socialsDiv {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  padding-left: 50px;
+  padding-bottom: 50px;
+}
+.socialButton {
+  width: 25px;
+  height: 25px;
+  margin: 4px;
+}
```

Comment:
sorry I should've mentioned this last PR too but CSS classes should be in [kebab case](https://wprock.fr/en/blog/conventions-nommage-programmation/), i.e. `social-button` instead of `socialButton`